engine/samples/imgui/page.md

```diff
@@ -0,0 +1,27 @@
+# ImGui {#examples-engine-imgui}
+
+@brief Using the @ref imgui-plugin plugin.
+
+The ImGui plugin allows you to integrate the [Dear ImGui](https://github.com/ocornut/imgui) library with CUBOS. engine, making it quick, straightforward, and effortless to create user interfaces and interactive debugging tools within your applications.
```

Comment:
very minor nitpick, but I think it sounds better if we write it either `with Cubos., making it ...` or `with the CUBOS. engine, making it...`